doc/source/tutorial/linalg.rst

```diff
@@ -402,7 +400,7 @@ Generalized inverse
 ^^^^^^^^^^^^^^^^^^^
 
 The generalized inverse is calculated using the command
-:obj:`linalg.pinv` or :obj:`linalg.pinv2`. These two commands differ
+:obj:`linalg.pinv`. These two commands differ
 in how they compute the generalized inverse. The first uses the
 linalg.lstsq algorithm, while the second uses singular value
 decomposition. Let :math:`\mathbf{A}` be an :math:`M\times N` matrix,
```

Comment:
This still contains an explanation for `pinv2`

Reply:
Ah apparently I forgot to read.